code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -409,7 +409,6 @@
 	new /obj/item/soap(src)
 	new /obj/item/grenade/chem_grenade/cleaner(src)
 	new /obj/item/grenade/chem_grenade/cleaner(src)
-	new /obj/item/melee/flyswatter(src)
```

Comment:
Weird that this is causing an overflow. UI Inventory rows have 7 spaces iirc and there are 7 items here, so it should fit just fine.

Reply:
`storage_slots = 6`

This forces it to have 6 so it overflows.